test/util/e2e.go

```diff
@@ -308,3 +309,19 @@ func GetKuberayTestImage() string {
 	gomega.Expect(found).To(gomega.BeTrue())
 	return kuberayTestImage
 }
+
+func CreateNamespaceWithLog(ctx context.Context, k8sClient client.Client, nsName string) *corev1.Namespace {
+	ginkgo.GinkgoHelper()
+	ns := utiltesting.MakeNamespace(nsName)
+	gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
```

Comment:
```suggestion
	gomega.ExpectWithOffset(k8sClient.Create(ctx, ns), 1).To(gomega.Succeed())
```

@mszadkow Could you use offset?

Reply:
ginkgo.GinkgoHelper() does that, at least that's how I understand it